File: app/src/main/java/com/pranav/java/ide/ui/treeview/model/TreeJavaFile.java
package com.pranav.java.ide.ui.treeview.model;

import android.content.Context;
import android.graphics.drawable.Drawable;

import androidx.appcompat.content.res.AppCompatResources;

import com.pranav.java.ide.R;

import java.io.File;

public class TreeJavaFile extends com.pranav.java.ide.ui.treeview.file.TreeFile {

    public TreeJavaFile(File file) {
        super(file);
    }

    @Override
    public Drawable getIcon(Context context) {
        return super.getIcon(context);
    }
}

File: app/src/main/java/com/pranav/java/ide/ui/treeview/model/TreeJavaFile.kt
package com.pranav.java.ide.ui.treeview.model

import android.content.Context
import android.graphics.drawable.Drawable

import java.io.File

class TreeJavaFile(file: File) : com.pranav.java.ide.ui.treeview.file.TreeFile(file) {

    override fun getIcon(context: Context) : Drawable? {
        return super.getIcon(context)
    }
}
